testsuite/tests/kuadrant/authorino/identity/rhsso/test_auth_credentials.py

```diff
@@ -12,33 +11,45 @@ def credentials(request):
 
 
 @pytest.fixture(scope="module")
-def authorization(rhsso, openshift, blame, route, module_label, credentials):
-    """Add RHSSO identity to AuthConfig"""
-    authorization = AuthConfig.create_instance(openshift, blame("ac"), route, labels={"testRun": module_label})
+def authorization(authorization, rhsso, credentials):
+    """Add RHSSO identity to Authorization"""
+    authorization.identity.clear_all()
```

Comment:
You don't need to clear identities here. `authorization` from rhsso conftest should be enough

Reply:
I do, due to parametrization